idsvr/helm-values.yaml

```diff
@@ -102,20 +154,35 @@ ingress:
     host: admin.curity.local
     secretName: curity-local-tls
 
-resources: {}
-  # We usually recommend not to specify default resources and to leave this as a conscious
-  # choice for the user. This also increases chances charts run on environments with little
-  # resources, such as Minikube. If you do want to specify resources, uncomment the following
-  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
-  # limits:
-  #   cpu: 100m
-  #   memory: 128Mi
-  # requests:
-  #   cpu: 100m
-  #   memory: 128Mi
+resources:
```

Comment:
The comment now does not reflect what is in the yaml.

Reply:
put back default config - accidental checkin